### PR TITLE
Add a CONTRIBUTING file

### PR DESCRIPTION
**Pull request by jiayliu**
Do we need a AUTHORS file?

### CODE REVIEW

**Comment by samdutton** [comment]
I also created a [Developer's Guide](https://docs.google.com/document/d/1tn1t6LW2ffzGuYTK3366w1fhTkkzsSvHsBnOHoDfRzY/edit#heading=h.e3366rrgmkdk), with some additions from others – but never made this public.

In some ways a Google doc can work better than a CONTRIBUTING file: in some ways it's easier to edit and comment on.

File: CONTRIBUTING [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+best way to ensure that future contributors do not break your code
+accidentally.
+
+
+To change the WebRTC source, you must submit a pull request in GitHub.
+See the GitHub documentation here:
+
+https://help.github.com/categories/collaborating/
+
+WebRTC developers monitor outstanding pull requests. They may request changes on
```
Comment (samdutton) [inline CONTRIBUTING]: 'changes on' => 'changes to'

File: CONTRIBUTING [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+
+In all cases, contributors must sign a contributor license agreement,
+either for an individual or corporation, before a patch can be accepted.
+Please fill out the agreement for an individual or a
+corporation, as appropriate.
+
+https://developers.google.com/open-source/cla/individual
+https://developers.google.com/open-source/cla/corporate
+
+If you or your organization is not listed there already, you should
```
Comment (juberti) [inline CONTRIBUTING]: I don't think we use AUTHORS in the samples repo.

**Comment by juberti** [comment]
LG with removal of AUTHORS ref. Linking Sam's guide is probably a good idea too.

**Comment by samdutton** [comment]
LGTM. Will merge and add link to my guide.

**Pull request merged**